publish/samplesite/scripts/controls/ribbon/Ribbon.tsx

```diff
@@ -30,6 +31,9 @@ export default class Ribbon extends React.Component<RibbonProps, {}> {
                         onClicked={this.onButtonClicked}
                     />
                 ))}
+                <button onClick={this.onRotateImage} className={styles.textButton}>
+                    RotateImage

```

Comment:
(Low pri)
if you have got a SVG file from Philip, let's use it in the demo site, and you can add a drop down menu to provide more rotation choices.

Reply:
Philip opened a work item for get an svg for this in ADO